Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2022 stripe-ios. All rights reserved.
 //
 
-@_spi(STP) import StripePaymentSheet
+@_spi(STP) @_spi(EmbeddedPaymentElementPrivateBeta) import StripePaymentSheet
```

Comment:
Can we remove (STP)?  Leaving it risks not noticing if we break visibility of an API.